src/dataExplorer/components/ResultsContext.tsx

```diff
@@ -58,6 +65,16 @@ export const ResultsProvider: FC = ({children}) => {
     } as SimpleTableViewProperties,
   })
 
+  const handleSetResult = useCallback(
+    (newResult: Partial<FluxResult>) => {
+      setResult({
```

Comment:
`setResult()` is called throughout codebase with the fromFlux parse function outcome. Don't allow it to overwrite the metadata.